CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -1,6 +1,18 @@
 # Everything beyond this point will be determined relatively
 # from this path.
 install_prefix="@CMAKE_INSTALL_PREFIX@"
+# Reset previous uvcdat env messing up
+source @CMAKE_INSTALL_PREFIX@/bin/reset_runtime.sh
+
+# Saves what we will mess with
+for v in PATH LD_LIBRARY_PATH DYLD_LIBRARY_PATH PYTHONPATH OPAL_PREFIX LIBOVERLAY_SCROLLBAR ; do
+   tmp="${v}"
+    if [ -n "${!tmp}" ] ; then
```

Comment:
Right, I understand that. What I meant is that it would be a lot of scripts that we'd have to support if we were to support each shell. So my solution was that either have the user use bash.

But seems like you would like to support all shells of interest.